Druid/DRIRGenerator.class.st

```diff
@@ -311,6 +311,8 @@ DRIRGenerator >> initializeSpecialCases [
 	
 	
 	specialCases at: #longAt:put: put: #interpretMachineWordAtPutWith:.
+	specialCases at: #unsignedLong64At:put: put: #interpretMachineWordAtPutWith:.
+	specialCases at: #unsignedLongAt:put: put: #interpretMachineWordAtPutWith:.
```

Comment:
Ah we should do a pass on all the loads and stores after all the branches are integrated :)